platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -375,13 +377,13 @@ def get_transceiver_info(self):
         except (ValueError, TypeError) : pass
 
         # Attempt ext_media read
-        if ext_media_module is not None:
-            ext_media_dict = ext_media_module.get_ext_media_info(self)
-            for key in ext_media_dict:
-                value = ext_media_dict[key]
-                if value in [None, 'None', 'none','n/a', '']:
-                    value = 'N/A'
-                transceiver_info_dict[key] = str(value)
+#        if ext_media_module is not None:
+#            ext_media_dict = ext_media_module.get_ext_media_info(self)
+#            for key in ext_media_dict:
+#                value = ext_media_dict[key]
+#                if value in [None, 'None', 'none','n/a', '']:
+#                    value = 'N/A'
+#                transceiver_info_dict[key] = str(value)
```

Comment:
Prefer to delete unused code rather than comment it out, unless there is a very good reason otherwise.

Reply:
ext_media is pending in review/ addressing comments. Once this is committed we will enable this code. Let us know if this needs to be deleted.

Reply:
No, as long as you intend to uncomment it soon, that's a good enough reason for me. :)